app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -37,8 +38,8 @@
     private final String owner = "owner";
     private final int sequenceNumber = 1;
     private final String ownHash = "ownHash";
-    private final String previousHashChain = "previousHashChain";
-    private final String previousHashSender = "previousHashSender";
+    private final String previousHashChain = "root";
+    private final String previousHashSender = "root";
```

Comment:
It was changed so that my backtracking function knows when it has reached the root block. Now my  back trace function does not look at these values any more so this is just something that remain changed. Fixing it right now.